enterprise/templates/enterprise/admin/setup_auth_org_id.html

```diff
@@ -0,0 +1,54 @@
+{% extends "admin/base_site.html" %}
+{% load i18n static admin_urls %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+  <a href="{% url 'admin:index' %}">{% trans "Home" %}</a>
+  &rsaquo; <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a>
+  &rsaquo; {% if has_change_permission %}
+    <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a>
+  {% else %}
+    {{ opts.verbose_name_plural|capfirst }}
+  {% endif %}
+  &rsaquo; {% if has_change_permission %}
+    <a href="{% url opts|admin_urlname:'change' enterprise_customer.uuid %}">
+      {{ enterprise_customer|truncatewords:"18" }}
+    </a>
+  {% else %}
+    {{ enterprise_customer|capfirst }}
+  {% endif %}
+  &rsaquo;
+  {% trans "Setup Auth org id" %}
+</div>
+{% endblock %}
+
+{% block content %}
+<div id="content-main">
+  <div class="forms-panel">
+    <h1>{% trans "Setup Auth org id" %}</h1>
+    <p>
+      This action is required for customers who will have learners in executive education courses. Setting up the
+      Auth org id will enable the enterprise's learners to take Exec ed or OCM courses using the same set of login
+      credentials. Clicking the button below will facilitate the necessary steps with our identity vendor, Auth0, and
+      will overwrite any value that may already be in the "Auth org id" field.
+    </p>
+    <form action="" method="post" id="setup-auth-org-id-form">
+      {% csrf_token %}
+      <div style="display:flex";>
+          <div style="vertical-align:top">
+              <input type="hidden" name="post" value="submit" />
+              <input type="hidden" name="action" value="submit" />
+              <input type="submit" value="{% trans "Setup Auth org id" %}" />
```

Comment:
[consider] May be helpful to make this button more prominent than the "Cancel" button to the right. Given this button sort of acts like a save button for the `EnterpriseCustomer`, I might recommend applying the `var(--default-button-bg)` CSS variable as the button background (same background color as Django's default "Save"):

<img width="239" alt="image" src="https://github.com/openedx/edx-enterprise/assets/2828721/7355840d-7f3b-4da5-a54a-80db40f68f16">

```suggestion
              <input type="submit" value="{% trans "Setup Auth org id" %}" style="background: var(--default-button-bg);" />
```